src/posts/lucky_0.29.cr

```diff
@@ -0,0 +1,192 @@
+class Lucky029Release < BasePost
+  title "Happy Holidays and welcome Lucky 0.29"
+  slug "lucky-0_29-release"
+
+  def published_at : Time
+    Time.utc(year: 2021, month: 12, day: 8)
+  end
+
+  def summary : String
+    <<-TEXT
+    The Lucky 0.29 release brings us many more
+    quality of life updates, bug fixes, and a
+    few new features to really make your app
+    solid.
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    We've done it again! Lucky 0.29.0 is now out, and a few apps
+    in the wild have already upgraded.
```

Comment:
```suggestion
    Lucky 0.29.0 is now out, and a few apps
    in the wild have already upgraded!
```

This made me chuckle, but it feels self-congratulatory in a way that might be off putting. What do you think?